Source/JavaScriptCore/parser/Lexer.cpp

```diff
@@ -802,24 +802,25 @@ static ALWAYS_INLINE bool cannotBeIdentPartOrEscapeStart(UChar c)
 
 
 template<>
-ALWAYS_INLINE UChar32 Lexer<LChar>::currentCodePoint() const
+ALWAYS_INLINE char32_t Lexer<LChar>::currentCodePoint() const
 {
     return m_current;
 }
 
+constexpr char32_t errorCodePoint = static_cast<char32_t>(U_SENTINEL);
```

Comment:
Would something like unicodeSentinelCodePoint be a better name for this?

Reply:
It depends if we want to name this based on where the value comes from or what it’s used for. The value comes from the ICU library, which defines a single sentinel value. But what this is used for here is to indicate an error. Either name is OK with me.

There are actually tons of non-character values, and any of them could be used to indicate an error here, so there’s no strong need to use `U_SENTINEL` here.

Reply:
I left this named `errorCodePoint`, but I stopped using `U_SENTINEL` for it. I also moved it to be a private class member.

Reply:
I don't see stopping using U_SENTINEL in the latest PR.  It's probably fine either way.

Reply:
Final one had this change.